browser/js/movies/movie-item/movie-item.html

```diff
@@ -6,6 +6,8 @@
 				<div class="list-group-item"><strong>{{movie.title}}</strong></div>
 				<div class="list-group-item">Category: <span ng-repeat="category in movie.category">{{category.name}} <span></div>
 				<div class="list-group-item">Rating: <span class="glyphicon glyphicon-star"></span><span class="glyphicon glyphicon-star"></span><span class="glyphicon glyphicon-star"></span></div>
+				<div ui-sref="addReview({reviewId: review._id})">Read reviews</div>
+				<div>Watch trailer</div>
```

Comment:
where does this ui-sref go? addReview doesn't take a review._id.

Also, Read Review probably shouldn't point to a state called addReview.